templatetags/htk_tags.py

```diff
@@ -285,6 +273,20 @@ def is_editable_by_context_user(context, obj):
         is_editable = False
     return is_editable
 
+@register.simple_tag(takes_context=True)
+def has_permission(context, permission_codename):
+    request = context.get('request', {}).get('request', None)
+    if request and request.user:
+        user = request.user
+        permission = Permission.objects.get(codename=permission_codename)
```

Comment:
what if you pass in a garbage codename, and it doesn't exist?

seems circuitous and wasteful to get the permission object, only to extract the `app_label`

would like a better way

Reply:
is it possible to pass in multiple parameters? in this case, also pass in the `app_label`

or try to get `app_label` from somewhere else, if it's available

Reply:
I get this when I search for a Permission with non-existent codename
![image](https://user-images.githubusercontent.com/5131486/44497437-1f61a700-a62e-11e8-9d52-81ad6d52a742.png)

I will wrap this in a try/except

Reply:
No, the docs say to pass in a string `app_label.permission_codename`.